addons/website_event/views/event_templates_list.xml

```diff
@@ -352,9 +352,7 @@
                         <!-- Location -->
                         <t t-set="has_city_or_country" t-value="event.address_id.sudo().city or event.address_id.sudo().country_id"/>
                         <div t-if="not event.address_id or has_city_or_country" class="d-flex align-items-center">
-                            <i class="fa fa-map-marker me-2" title="Location"/>
-                            <small t-if="event.address_id" class="o_not_editable fw-bold" itemprop="location" t-out="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'no_marker': 'true'}"/>
-                            <small t-else="" class="o_not_editable fw-bold" itemprop="location">Online event</small>
+                            <small class="o_not_editable fw-bold" itemprop="location" t-out="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'null_text': 'Online event'}"/>
```

Comment:
Ok then to display the country in case we don't have the city, we could add `country_id` in the list of fields here